Backendless/Persistence/LineString.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace BackendlessAPI
+{
+  public class LineString : Geometry
+  {
+    public const String GEOJSON_TYPE = "LineString";
+    public static String WKT_TYPE = GEOJSON_TYPE.ToUpper();
+
+    private List<Point> points;
+
+    public LineString( List<Point> points) : this( points, SpatialReferenceSystem.DEFAULT )
+    {
+    }
+
+    public LineString(List<Point> points, ReferenceSystemEnum srs ) : base( srs )
+    {
+      this.points = new List<Point>( points );
+    }
+
+    public List<Point> GetPoints()
+    {
+      return new List<Point>( this.points );
+    }
+
+    public LineString SetPoints( List<Point> points )
+    {
+      this.points.Clear();
+      this.points.AddRange( points );
+      return this;
+    }
+
+    public override String GetGeoJSONType()
+    {
+      return LineString.GEOJSON_TYPE;
+    }
+
+    public override String GetWKTType()
+    {
+      return LineString.WKT_TYPE;
+    }
+
+    internal override String JSONCoordinatePairs()
+    {
+      StringBuilder sb = new StringBuilder();
+      sb.Append( '[' );
+      foreach ( Point p in this.GetPoints() )
+        sb.Append( p.JSONCoordinatePairs() ).Append( ',' );
+
+      sb.AppendLine( "]" );
+      return sb.ToString();
+    }
+
+    internal override String WKTCoordinatePairs()
+    {
+      StringBuilder sb = new StringBuilder();
+      foreach ( Point p in this.GetPoints() )
+      {
+        sb.Append( p.WKTCoordinatePairs() );
+
+        if (points[points.Count - 1] != p)
```

Comment:
here you use `points` directory, but in the loop you use `GetPoints()` this is bad style, either use encapsulation or don't, but be consistent.